src/views/dasboard/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import classnames from 'classnames'
+import { useDispatch, useSelector } from 'react-redux'
+
+import Header from '_components/header'
+import { CHANGE_NAME, LOGGED_AS, USER_ID } from '_utils/dashboard'
+import { getUserSelector } from '_modules/user/selectors'
+import Input from '_components/input'
+import Button, { ButtonTheme } from '_components/button'
+import { getUser, updateUser } from '_modules/user/actions'
+import Spinner, { SpinnerTheme } from '_components/spinner'
+
+import styles from './styles.css'
+
+const ALLOW_IMAGES_EXTENSIONS = ['.jpeg', '.jpg', '.png']
+
+function getBase64(file) {
+  return new Promise((resolve, reject) => {
+    const reader = new FileReader()
+    reader.readAsDataURL(file)
+    reader.onload = () => resolve(reader.result)
+    reader.onerror = error => reject(error)
+  })
+}
+
+const Dashboard = () => {
+  const user = useSelector(getUserSelector)
+  const [userNewName, setUserNewName] = useState('')
+  const [isLoading, setIsLoading] = useState(false)
+
+  const dispatch = useDispatch()
+
+  const handlePictureChange = useCallback(
+    event => {
+      const file = event.target.files[0]
+      getBase64(file).then(data => {
+        dispatch(updateUser({ avatar: data }, user.id))
+      })
+    },
+    [dispatch, user.id]
+  )
+
+  const handleChangeAvatar = useCallback(() => {
+    const inputFile = document.createElement('input')
+    inputFile.value = ''
+    inputFile.type = 'file'
+    inputFile.accept = ALLOW_IMAGES_EXTENSIONS
+    inputFile.onchange = handlePictureChange
+    inputFile.click()
+  }, [handlePictureChange])
+
+  const handleOnChange = (event, state) => {
+    const { value } = event.target
+
+    state(value)
+  }
+
+  const handleSubmit = async event => {
+    event.preventDefault()
+    setIsLoading(true)
+
+    if (userNewName) {
+      await dispatch(updateUser({ name: userNewName }, user.id))
+    }
+    setIsLoading(false)
+  }
+
+  useEffect(() => {
+    setIsLoading(true)
+    dispatch(getUser())
+    setIsLoading(false)
+  }, [dispatch])
+
+  return (
+    <div className={styles['dashboard-container']}>
+      <Header />
+
+      {isLoading ? (
+        <div className={styles['user-loading']}>
+          <Spinner color="black" size={SpinnerTheme.X_LARGE} />
+        </div>
+      ) : (
+        <>
+          <section className={styles['dashboard-content']}>
+            <h1 className={classnames(styles['title-bold-information'], styles['logged-as'])}>
+              {LOGGED_AS}
+            </h1>
```

Comment:
Headings are supposed to be titles for sections.
"You're logged as" is not a title at all.
Think about the structure of the view you have and ignore the styling when deciding to use headings. It's not just because the text is bold or has a big font size that it will be a title.
This view is more of a User Profile. This could have been the heading 1 of the page. If it's not on the design, you can always use hidden headings to keep semantic correct.